ether-wake.c

```diff
@@ -109,40 +109,56 @@ static int get_nfqueue_num(const char *optarg);
 int main(int argc, char *argv[])
 {
 	char *ifname = "eth0";
-	int one = 1;				/* True, for socket options. */
-	int errflag = 0, nfqueue_errflag = 0, verbose = 0, do_version = 0;
+	int one = 1; /* True, for socket options. */
+	int errflag =0, nfqueue_errflag = 0, verbose = 0, do_version = 0;
```

Comment:
Could you fix the formatting here?

Reply:
In general, I wonder if this file maybe shouldn't be reformatted so that potential backports from ether-wake are easier.

Reply:
Sure I can revert it, but I personally believe that the whole project should have a coherent format, also when I edit the file, I'll not be able to automatically format my code.

Reply:
I agree that consistent coding style, and therewith also supporting auto-format, is a good thing. For 'external' code though, which ether-wake.c mostly is, I prefer to not touch the formatting to run diffs against any newer versions.

Reply:
Well but what should I do with my hold feature, which waits to accept packages until it received a ping from the host? Currently I've changed ether-wake.c for that.

Reply:
You can of course add stuff to the file, potentially you could add the ping functionality to a new ping.c file. In general, the ping feature doesn't depend on the MNL changes, does it?

Reply:
No it does not.